app/lib/switches.rb

```diff
@@ -288,7 +288,7 @@ def switches
   def globally_denied_switches
     [
       account.master_on_premises? ? :finance : nil,
-      ThreeScale.config.onpremises ? :end_users : nil
+      nil
```

Comment:
remove the line directly :smile: `compact` (the next line) removes all the `nil` values of the array :smile: